templates/base/src/utils/Logger/logger.js

```diff
@@ -0,0 +1,65 @@
+
+import fetch from 'isomorphic-unfetch';
+
+/**
+ * USAGE: ->
+ * import Logger from '../utils/Logger';
+ *  call as :-
+ * Logger.error({ message: 'Here is the error' });
+ */
+
+
+/**
+ * Will be modified once the post method is configured on server
+ */
+const submitLogMsg = (msg) => {
+  const { message } = msg;
+  fetch('/mock/logs.json', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      message
+    })
+  }).catch((err) => { console.log('log-err=====>>>', err) })
+}
+
+const callLog = ({ logLevels, logLevel, msgObj }) => {
+  const isDev = process.env.NODE_ENV !== 'production';
+  const isServer = !process.browser;
+  const isLogDisabled = !!process.env.LOG_DISABLED;
+  const currentLogLevel = process.env.LOG_LEVEL || (isDev ? 'debug' : 'info');
+  const [msg] = msgObj;
```

Comment:
@all9planets before sending the log into API call we should merge the log data into a uniform structure. 

Consider following as default minimum structure: 

```javascript/json
{
  "appName": "", //Mandatory
  "logLevel": "info | error"
  "browser": { //Mandatory for browser side
      "userAgent": "",
      "location": window.location.href,
      "host": window.location.host
    },
   "userInfo": { //Optional and to be updated by application
   },
   "event": { //Optional and to be updated by application through log call
      "name": "",
      "id": "",
       "module": "", 
        "type": "load|interaction|redirect" 
       "data": {}
    },
    error: {  //Optional and to be updated by application through log call
      code: "",
      message: "",
      "operationName": ""
    },
    service: { //Optional and to be updated by application through log call
       name: "",
       path: ""
    }
}
```

Reply:
done